ci_utils/check_commit_signature.py

```diff
@@ -0,0 +1,24 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) AGPLv3 2016-2021 Scille SAS
```

Comment:
I'd rather have a `.github/scripts` folder instead of this `ci_utils`: one less entry in this too crowded root folder ;-)

(besidet that's [what is done in the doc example](https://docs.github.com/en/actions/learn-github-actions/essential-features-of-github-actions#adding-scripts-to-your-workflow) ^^)

Reply:
Add moving ci scripts out of the config file is totally the right move for readability ! 😃

Reply:
done